### ISSUE TITLE
Update Dialect List

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-translations#48]
## Update Dialect List

The dialect list in the [`locale/dialect-list.yml`](https://github.com/legesher/legesher-translations/blob/master/locale/dialect-list.yml). It would be extremely helpful to update this list with the content found in the `English Language Name` column so we can properly refer to them in the issues. 

We've used [this](https://meta.wikimedia.org/wiki/Template:List_of_language_names_ordered_by_code) as a reference for language dialects.

Please update the `dialect-list.yml` file to reflect the following style for each dialect:

```yml
Amharic:
  abbreviation: "am"
  localName: "አማርኛ"
```

**Comment by jonnyfluckey** [comment]
I'm willing to give this one a try!

**Comment by madiedgar** [comment]
Awesome @jonnyfluckey! That would be amazing!! 👍

**Comment by jonnyfluckey** [comment]
#55 
Thanks for the opportunity to work on this. I would love to find more ways to contribute, as this is a great cause

**Comment by madiedgar** [comment]
Thank you!! 🎉 & we would love that @jonnyfluckey! Is there something specific you'd like to grow in / language you'd like to focus on? I can keep you in mind for future issues 😄

**Comment by jonnyfluckey** [comment]
@madiedgar ,

I'm pretty open to learning new things. My front-end strengths are in React, but I have also dabbled in Angular and am interested in learning that and Typescript more.

Back-end, I have dabbled in basic Node servers with Express,  but would love the chance to learn more about it and dig deeper into some of those concepts.

Either way, I'm open to contributing in any way that I can :)

Thanks again!
